scripts/convert_geojson_to_legacyjson.py

```diff
@@ -27,8 +54,28 @@ def convert_json_source(source):
 
     return converted
 
+try:
+    opts, args = getopt.getopt(sys.argv[1:], "btr",["bbox","tmsonly","removepolygons"])
```

Comment:
Can we use Python's [`argparse` library](https://docs.python.org/3/library/argparse.html) instead?